packages/shared/src/components/cards/PostCard.tsx

```diff
@@ -66,86 +65,109 @@ export const PostCard = forwardRef(function PostCard(
     children,
     showImage = true,
     style,
+    insaneMode,
+    onReadArticleClick,
     ...props
   }: PostCardProps,
   ref: Ref<HTMLElement>,
 ): ReactElement {
-  const [selectedComment, setSelectedComment] = useState<Comment>();
+  const { postCardVersion, postEngagementNonClickable } =
+    useContext(FeaturesContext);
+  const isV1 = postCardVersion === 'v1';
+  const Containter = useMemo(
+    () =>
+      classed(
+        'div',
+        'relative flex flex-1',
+        postCardVersion === 'v1' ? 'flex-col' : 'flex-col-reverse',
+      ),
+    [postCardVersion],
+  );
   const { trending } = post;
-
-  const customStyle =
-    selectedComment && !showImage ? { minHeight: '15.125rem' } : {};
+  const customStyle = !showImage ? { minHeight: '15.125rem' } : {};
   const card = (
     <Card
       {...props}
-      className={getPostClassNames(post, selectedComment, className)}
+      className={getPostClassNames(post, className)}
       style={{ ...style, ...customStyle }}
       ref={ref}
     >
       <PostLink post={post} openNewTab={openNewTab} onLinkClick={onLinkClick} />
```

Comment:
We should use `PostLink` only when we don't open the article modal by default. Otherwise, there's no need to have background anchor element. The only button that opens that article and it's an actual link is the "read article" button

Reply:
Yes it is for opening a link, though it can be reused to instead make the whole card clickable. But we can make a separate element for it if necessary. Or make it a button element instead if the user has post by modal.

Reply:
Yes having a link in the background just to have something clickable is not the right thing from a semantic HTML perspective. You can use a button as you mentioned

Reply:
I agree. Let's turn this one into a button only when necessary.